math/mathcore/inc/Math/MinimizerOptions.h

```diff
@@ -28,23 +28,107 @@ class IOptions;
     Minimizer options
 
     @ingroup MultiMin
+
+    Class defining the options for the minimizer. 
+    It contains also static methods for setting the default Minimizer option values
+    that will be used by default by all Minimizer instances.
+    To see the current default options do:
+
+         ROOT::Math::MinimizerOptions::PrintDefault();
+
 */
 class MinimizerOptions {
 
 public:
 
    // static methods for setting and retrieving the default options
 
+   /// Set the the default Minimizer type and corresponding algorithms 
```

Comment:
```suggestion
   /// Set the the default Minimizer type and corresponding algorithms.
```
I think doxygen needs a dot after the function short description.